.env

```diff
@@ -13,13 +13,19 @@ NGINX_PORT_HOST=81
 NGINX_SSLPORT_HOST=8443
 
 # CKAN databases
-POSTGRES_USER=ckan
-POSTGRES_PASSWORD=ckan
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=admindbpassword
+POSTGRES_DB=admindb
```

Comment:
This is probably better as just postgres, it will be generated automatically and that's what people expect when using postgres. It's also generally going to be the same name as the superuser, at least by convention.